src/crafting.cpp

```diff
@@ -607,15 +607,11 @@ void player::complete_craft()
             }
         }
 
-        // Don't store components for things made by charges,
-        // don't store components for things that can't be uncrafted.
-        if( recipe_dictionary::get_uncraft( making->result() ) && !newit.count_by_charges() ) {
-            // Setting this for items counted by charges gives only problems:
-            // those items are automatically merged everywhere (map/vehicle/inventory),
-            // which would either loose this information or merge it somehow.
+        if( !newit.has_flag( "ALWAYS_STACKS" ) ) {
```

Comment:
This is a bit of a misnomer. While it does affect stacking, it does more than that. And items of the same type will still not stack if other properties don't match up (e.g. temperature/rotten state), so it's not "*always* stacking".